tests/tools/test_create_type.py

```diff
@@ -0,0 +1,75 @@
+from textwrap import dedent
+
+import strawberry
+from strawberry.tools import create_type
+
+
+def test_create_basic_type():
+    @strawberry.field
+    def name() -> str:
+        return "foo"
+
+    MyType = create_type("MyType", [name])
+    definition = MyType._type_definition
+
+    assert len(definition.fields) == 1
+
+    assert definition.fields[0].graphql_name == "name"
+    assert definition.fields[0].type == str
+
+
+def test_create_mutation_type():
+    @strawberry.type
+    class User:
+        username: str
+
+    @strawberry.mutation
```

Comment:
I would add another test that checks the parameters (e.g permission_classes=[X]) passed to  `@strawberry.mutation()` are not lost

(this is coming from experience where in a previous version of the create_type function I had this bug :D)

Reply:
Good idea!